plugins/TwoFactorAuth/tests/UI/TwoFactorAuth_spec.js

```diff
@@ -20,13 +20,19 @@ describe("TwoFactorAuth", function () {
     async function selectModalButton(button)
     {
         await (await page.jQuery('.modal.open .modal-footer a:contains('+button+')')).click();
+        await page.waitForNetworkIdle();
     }
 
     async function loginUser(username, doAuth)
     {
         // make sure to log out previous session
         await page.goto(logoutUrl);
 
+        var cookies = await page.cookies();
+        cookies.forEach(cookie => {
+            page.deleteCookie(cookie);
```

Comment:
It seems in tests logging out doesn't kill cookies, and so the some session data is still available. That caused some 2fa tests to fail, as the password verify was still stored and not triggered again